.travis.yml

```diff
@@ -33,5 +33,8 @@ matrix:
       before_script:
         - export BINDIR=$(pwd) && curl -H 'Cache-Control:no-cache' https://raw.githubusercontent.com/fossas/fossa-cli/master/install.sh | bash
       script:
+        - ls sddfssdfsdf # test
+        - [[ -z "$FOSSA_API_KEY" ]] && [[ "$TRAVIS_PULL_REQUEST" != "false" ]] && travis_terminate 0 # pass on PRs from forks that do not get FOSSA_API_KEY
+        - echo test && travis_terminate 0 # test
```

Comment:
and this?

Reply:
yeah, still testing if it works correctly :)